lib/src/modules/status.cc

```diff
@@ -134,63 +143,102 @@ ActionOutcome Status::callAction(const ActionRequest& request) {
     LOG_DEBUG("Retrieving results for job %1% from %2%",
               t_id, results_dir_path.string());
 
-    auto metadata_file = (results_dir_path / "metadata").string();
+    bool running_by_pid { false };
+    bool not_running_by_pid { false };
+    auto pid_file = (results_dir_path / "pid").string();
```

Comment:
You're inside the ActionOutcome namespace, is it really confusing?
I find it ugly to have such redundancy ("action" - "action") in the instance variables names: ActionOutcome::action_pid_file.
Besides, that's exactly the name of the files of the directory: metadata, pid, stdout, stderr.

I'm happy to change if you really think it's confusing.